packages/react-components/src/components/VideoGallery/DefaultLayout.tsx

```diff
@@ -80,7 +80,7 @@ export const DefaultLayout = (props: DefaultLayoutProps): JSX.Element => {
    * this case when those components are here
    */
   const [indexesToRender, setIndexesToRender] = useState<number[]>([
-    ...Array(maxRemoteVideoStreams - activeVideoStreams).keys()
+    ...Array(Math.max(maxRemoteVideoStreams - activeVideoStreams, 1)).keys()
```

Comment:
Not super clear why there is always atleast 1 index here, can you add a comment

Reply:
Added the  comment and the number changed to 4. In case the current number of videos rendered is greater than the max, we will render just 4